include/lsst/jointcal/CcdImage.h

```diff
@@ -72,6 +72,13 @@ class CcdImage {
         getWholeCatalog().copyTo(getCatalogForFit());
     }
 
+    /**
+     * Count the number of valid measured and reference stars that fall within this ccdImage.
+     *
+     * @return Number of (measured, reference) stars in the image.
```

Comment:
please add a few words explaining whether reference stars that are associated with an invalid measured star are counted or excluded (with some sort of justification, if practical; it may not be worth the trouble, since this count is only intended as a simple diagnostic measure and it may not be clear yet -- or ever -- which option is preferable).